share/man/man9/PCI_DEV.9

```diff
@@ -0,0 +1,84 @@
+.\" Copyright (c) 2018 Lakhan Shiva Kamireddy <lakhan@FreeBSD.org>
+.\" Sponsored by: Google, Inc. (GSoC 2018)
+.\" Mentors: Warner Losh, Chuck Tuffli
+.\" All rights reserved.
```

Comment:
These three lines aren't needed. We don't document this stuff in the man pages. All Rights Reserved isn't useful anymore.